internal/manifest/v1beta1/custom_resource.go

```diff
@@ -96,3 +120,14 @@ func PreDeleteDeleteCR(
 	}
 	return nil
 }
+
+func GetCRDName(obj declarative.Object) string {
+	manifest := obj.(*v1beta2.Manifest)
+	if manifest.Spec.Resource != nil {
+		group := manifest.Spec.Resource.GroupVersionKind().Group
+		name := manifest.Spec.Resource.GroupVersionKind().Kind
+		return fmt.Sprintf("%ss.%s", strings.ToLower(name), group)
```

Comment:
consider making that plural `s` more visible for readability purposes, e.g: `fmt.Sprintf("%s.%s", getPlural(name), group)`